.github/workflows/check-examples.yml

```diff
@@ -27,6 +27,7 @@ jobs:
         base: ${{ github.ref }}
         filters: |
           examples:
+            - 'setup.py'
```

Comment:
We probably should check for any changes that are not only updating the docs or the the tests? (So include here pytket/* )?